prody/ensemble/ensemble.py

```diff
@@ -54,16 +54,16 @@ def __init__(self, title='Unknown'):
     def __repr__(self):
 
         if self._indices is None:
-            return ('<Ensemble: {0} ({1} conformations; {2} atoms)>'
-                    ).format(self._title, len(self), self._n_atoms)
+            return ('<{0}: {1} ({2} conformations; {3} atoms)>'
+                    ).format(self.__class__.__name__, self.getTitle(), len(self), self._n_atoms)
         else:
-            return ('<Ensemble: {0} ({1} conformations; selected {2} of '
-                    '{3} atoms)>').format(self._title, len(self),
+            return ('<{0}: {1} ({2} conformations; selected {3} of '
+                    '{4} atoms)>').format(self.__class__.__name__, self.getTitle(), len(self),
                                           self.numSelected(), self._n_atoms)
 
     def __str__(self):
 
-        return 'Ensemble {0}'.format(self._title)
+        return 'Ensemble {0}'.format(self.getTitle())
```

Comment:
It probably makes sense to do the same here as above with the `self.__class__.__name__` or to not do it above. I don't think we probably need it above because we have separate __repr__ and __str__ functions for PDBEnsemble class though.